src/Vex/RHI/RHIDescriptorPool.h

```diff
@@ -8,25 +10,7 @@
 namespace vex
 {
 
-struct BindlessHandle : public Handle<BindlessHandle>
-{
-    enum class Type : i8
-    {
-        None = -1,
-        UniformBuffer = 0,
-        StorageBuffer = 1,
-        Texture = 2,
-        StorageImage = 3,
-        Num = 4
-    } type = Type::None;
-
-    static BindlessHandle CreateHandle(u32 index, u8 generation, Type type)
-    {
-        BindlessHandle handle = Handle::CreateHandle(index, generation);
-        handle.type = type;
-        return handle;
-    }
-};
+using BindlessHandle = Handle<>;
```

Comment:
The reason I didn't use usings for the handle system was to allow for compile time validation on handle types (eg not being able to pass a BindlessHandle to a function taking in another handle type). Does this maintain this functionality?